bedrock/firefox/templates/firefox/new/desktop/download.html

```diff
@@ -44,6 +44,8 @@
 
 {% block content %}
 
+{% set show_mr2_browser_promo = switch("browser-mr2-promo") and variant == 'fx94' and ftl_has_messages('firefox-desktop-love-your-life', 'firefox-desktop-its-your-internet') %}
```

Comment:
+1 here, I like the way you created an easy variable for this variation to be used.